main.js

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable no-restricted-syntax */
+
+localStorage.books = JSON.stringify([]);
+
+function displayBooks() {
+  const container = document.getElementById('container');
+  container.innerHTML = '';
+
+  const books = JSON.parse(localStorage.books);
+  if (books.length === 0) {
+    container.innerHTML = 'Shelf is empty 🙄❗';
+  } else {
+    for (const book of books) {
+      const title = document.createElement('p');
+      const author = document.createElement('p');
+      const removeBtn = document.createElement('button');
+      removeBtn.setAttribute('data-id', books.indexOf(book));
+      removeBtn.innerText = 'Remove';
+      title.innerHTML = book.title;
+      author.innerHTML = book.author;
+      container.append(title, author, removeBtn);
+    }
+  }
+}
+
+function removeBook(id) {
+  const books = JSON.parse(localStorage.books);
+  books.splice(id, 1);
```

Comment:
- [ ] OPTIONAL
It will be a good practice to favor the use of array methods such as `slice` over `splice`, which doesn't mutate the array.

Reply:
Thanks for the tip @rloterh :ok_hand:
however, in that situation i wanted to mutate the original array to keep it up to date.